packages/driver/src/cypress/script_utils.js

```diff
@@ -38,9 +38,12 @@ const runScriptsFromUrls = (specWindow, scripts) => {
 // Supports either scripts as objects or as async import functions
 const runScripts = (specWindow, scripts) => {
   // if scripts contains at least one promise
-  if (scripts.length && typeof scripts[0].then === 'function') {
-    // merge the awaiting of the promises
-    return Bluebird.all(scripts)
+  if (scripts.length && typeof scripts[0] === 'function') {
+    // chain the loading promises
+    // NOTE: since in evalScripts, scripts are evaluated in order,
+    // we chose to respect this constraint here too.
+    // indeed _.each goes through the array in order
+    return scripts.reduce((prev, cur) => prev.then(cur), Promise.resolve())
```

Comment:
How does this work - don't we want to return an array, or is it fine to just eval them all here? This won't return anything anymore - the behaviour is changed, is this correct?

Reply:
This will return the promise of the last file to be imported. 
Since we are local it should not take long.

Reply:
You would use `Bluebird.mapSeries()` here if you want them serially executed and in the same order they came in the array.

Reply:
Replaced with Bluebird.each to make it clearer and clean.